lab1.js

```diff
@@ -117,8 +149,21 @@ var tooHungryDay;
  pondering protein supplements (the first day the average dips below 4
  meals)
 */
-
+var totalMeals = 0;
+var firstTimeHungry = true;
 // your code goes here
+for (var i = 0; i < mealsPerDay.length; i++) {
+  //calculates the average and prints the day and average
+  totalMeals += mealsPerDay[i];
+  var averageMeals = totalMeals / (i + 1);
+  console.log('Average meals/day for day ' + (i + 1) + ' = ' + averageMeals);
+  //if statement to check if average meals is below 4 and if its the first time
+  if (averageMeals < 4 && firstTimeHungry) {
```

Comment:
like the use of the firstTimeHungry flag here to check whether it's the first time the lion was hungry. In JavaScript, you could also use a break to end the loop after you find the first one.